src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] arrayMachine = new Machine[]{new Truck(), new Bulldozer(), new Excavator()};
 
+        for (Machine machine : arrayMachine) {
+            System.out.println("");
+            machine.doWork();
+            machine.stopWork();
+            System.out.println("");
```

Comment:
if you want to print empty line:
```suggestion
            System.out.println();
```